addons/point_of_sale/static/src/app/generic_components/orderline/orderline.xml

```diff
@@ -7,11 +7,11 @@
                 <div class="product-name d-inline-block flex-grow-1 fw-bolder pe-1 text-truncate">
                     <span class="text-wrap" t-esc="line.productName"/>
                     <t t-slot="product-name"/>
-                </div>
+            </div>
                 <div t-esc="line.price" class="price fw-bolder"/>
             </div>
             <ul class="info-list ms-2">
-                <li>
+                <li class="price-per-unit">
```

Comment:
I wasn't able to see the use of this because I didn't look at the tour methods. But I guess I trust you that it's used.

Reply:
yes, it's used :)